graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/ClientApiGenerator.kt

```diff
@@ -170,6 +170,22 @@ class ClientApiGenerator(private val config: CodeGenConfig, private val document
             }
```

Comment:
Thanks for the changes, they look good to me. Could you also make the equivalent changes to kotlin as well, since we'd like to maintain feature parity.

Reply:
Hi @srinivasankavitha, it looks like client classes are always generated in Java. I'm not seeing a Kotlin version of `ClientApiGenerator`.

Furthermore, I just tried the following configuration in a fresh Kotlin/Gradle project which generated Java client classes but Kotlin for everything else:

```gradle
generateJava {
    schemaPaths = ["${projectDir}/src/main/resources/schema.graphql"]
    packageName = 'com.example.packagename'
    generateClient = true
    language = 'kotlin'
}
```

Can you point me in the right direction?

Reply:
Actually, you're right. The client classes are generated in java since it was a substantial amount of code duplication that we wanted to avoid to support that. So never mind, kotlin classes are mainly for data classes.